.github/workflows/ci.yaml

```diff
@@ -39,7 +44,8 @@ jobs:
 
       - name: Run integration tests (with mock DB)
         env:
-          OPENAI_API_KEY: ${{ secrets.OPENAI_API_KEY }}
+          openai: ${{ secrets.OPENAI }}
+          claude: ${{ secrets.CLAUDE }}
```

Comment:
I recommend still using ```OPENAI_API_KEY: ${{ secrets.OPENAI_API_KEY }}``` or ```openai_token``` at least.
openai doesn't tell you what it is, it's an api key.